src/components/ListOfEmissionsData/ListOfEmissionsData.tsx

```diff
@@ -0,0 +1,92 @@
+import { useListData } from '../../hooks/useListData';
+import { List } from 'react-window';
+import { RowComponent } from '../RowComponent/RowComponent';
+import { ColumnModal } from '../ColumnModal/ColumnModal';
+import { ListControls } from '../ListControls/ListControls';
+import { ListHeader } from '../ListHeader/ListHeader';
+import './ListOfEmissionsData.css';
+import { memo, useMemo } from 'react';
+
+const MemoizedListControls = memo(ListControls);
+const MemoizedListHeader = memo(ListHeader);
+const MemoizedColumnModal = memo(ColumnModal);
+const MemoizedRowComponent = memo(RowComponent);
```

Comment:
Don't need  to  wrap this components again 
It is already memoized inside every file
[ListControls](https://github.com/novogran/ReactPerformance/blob/aad3d3560af9dd2f00d27759fb68732bb320fe05/src/components/ListControls/ListControls.tsx#L15)
[ListHeader](https://github.com/novogran/ReactPerformance/blob/aad3d3560af9dd2f00d27759fb68732bb320fe05/src/components/ListHeader/ListHeader.tsx#L11)
and so forth